google/cloud/internal/win32/parse_service_account_p12_file.cc

```diff
@@ -121,9 +112,7 @@ StatusOr<UniqueCertStore> OpenP12File(std::string const& source) {
 StatusOr<UniqueCertContext> GetCertificate(HCERTSTORE certstore,
                                            std::string const& source) {
   // Get the certificate from the store.
-  PCCERT_CONTEXT cert_raw = CertFindCertificateInStore(
```

Comment:
`CertFindCertificateInStore` supports searching with more complex criteria. We just want the first/only certificate in the store, and therefore can use the simpler `CertEnumCertificatesInStore`.